config/crd/bases/awx.ansible.com_awxs.yaml

```diff
@@ -1467,6 +1467,28 @@ spec:
                         type: string
                     type: object
                 type: object
+              rsyslog_resource_requirements:
```

Comment:
@rakesh561 I believe this just needs to be moved one space to the left, then it should be correct yaml syntax.